evdns.c

```diff
@@ -671,21 +675,18 @@ nameserver_probe_failed(struct nameserver *const ns) {
 		return;
 	}
 
-#define MAX_PROBE_TIMEOUT 3600
```

Comment:
We had pretty huge default

Reply:
yeah I think so, set the default value to 60s might be reasonable, should I change this?